### ISSUE TITLE
Infinite loop problem with push action (Windows, mingw32)

### ISSUE THREAD

**Issue by Jntz** [git-ftp/git-ftp#192]
Hello. I tried to sync remote and local server. I need to use ftpes protocol. Commands **init**, **catchup** and **show** working properly but somehow push -command struct in upload action.

This is my output what I get: 
```
Thu May  7 14:00:37 FLEDT 2015: CACert is ''.
Thu May  7 14:00:38 FLEDT 2015: Insecure is '1'.
Thu May  7 14:00:38 FLEDT 2015: Retrieving last commit from ftp://xxxxxx:***@
xxx.yyy.zzz/dddddd/.
######################################################################## 100,0%
Thu May  7 14:00:45 FLEDT 2015: Last deployed SHA1 for XXXX/YYYY/ is 2140ad83d87d106197285a1aa1cc129e0cc25639.
mktemp: No such file or directory
Thu May  7 14:00:46 FLEDT 2015: Having files to sync.
Thu May  7 14:00:46 FLEDT 2015: There are 1 files to sync:
Thu May  7 14:00:46 FLEDT 2015: [1 of 1] Buffered for upload 'test2.txt'.
Thu May  7 14:00:47 FLEDT 2015: Uploading ...
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
######################################################################## 100,0%
```
You can see that script upload a file 10 times. (I used verbose output) It upload one text file forever if I don't kill process

**Comment by mkllnk** [comment]
Hi,

there is one line that should not be there:

    mktemp: No such file or directory

That should be fixed in the latest development version. Can you try with that one again?

Are you using Windows or MacOS?

**Comment by Jntz** [comment]
I use Windows and MacOS. MacOS doesn't have any problem. I think bigger problem is unix-type commandline programs. I realised if I use push command it will push every location what I used. 

Example: First I try to push the project to remote directory "A" and it's caused infinite loop. Then I remove directory "A" in remote and create duplicate project (local) and init + push to remote location "B". And try again with location "C".

When I push to location C it's also push (really make init task) to location "B" and "A". I don't get same problem with MacOS so I concluded my unix commandline in Windows is the main problem. I think to close this issue.
